docs/rules/prop-types.md

```diff
@@ -159,8 +159,22 @@ As it aptly noticed in
 > Most components don't need `this.props.children`, so that makes it extra important
 to document `children` in the prop types.
 
-Generally, you should use `PropTypes.node` or static type `React.Node` for
-`children`. It accepts anything that can be rendered: numbers, strings, elements
```

Comment:
https://flow.org/en/docs/react/types/#toc-react-node is for Flow. `ReactNode` is for TypeScript.

We shouldn't remove the Flow instructions; let's keep both.

Reply:
This still needs the Flow instructions added.